gtsfm/utils/metrics.py

```diff
@@ -451,3 +453,70 @@ def compute_percentage_change(x: float, y: float) -> float:
         percentage change (may be positive or negative).
     """
     return (y - x) / (x + EPSILON) * 100
+
+
+def compute_psnr_after_downsampling(
+    original_point_cloud: np.ndarray, downsampled_point_cloud: np.ndarray
+) -> GtsfmMetric:
+    """Compute PSNR between original point cloud and downsampled point cloud
+    A larger PSNR shows that the downsampled point cloud consists better with the original point cloud
```

Comment:
not sure what you mean by `consists better` ?

Reply:
correlates with or corresponds to?